### PR TITLE
Define proper error page and error message

### PR DESCRIPTION
**Pull request by utpalBeehyv**
https://github.com/pucardotorg/dristi/issues/2151

## Summary by CodeRabbit

- **New Features**
	- Enhanced error component with improved styling and presentation.
	- Added contact information for support displayed during error events.

- **Bug Fixes**
	- Updated error message to provide clearer communication to users.

### CODE REVIEW

**Review by anirudh-0 — APPROVED** [review]

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/core/src/components/ErrorComponent.js [hunk 1]
```diff
@@ -1,10 +1,67 @@
...
+    fontSize: "16px",
+  },
+});
+
 const ErrorConfig = {
   error: {
-    imgUrl: `https://s3.ap-south-1.amazonaws.com/egov-qa-assets/error-image.png`,
-    infoMessage: "CORE_SOMETHING_WENT_WRONG",
+    imgUrl: `https://pucarfilestore.blob.core.windows.net/pucar-filestore/pg/pucar-assets/Frame.png`,
+    infoMessage: "ERROR_MESSAGE",
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/core/src/components/ErrorComponent.js:64]: No Need

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/core/src/components/ErrorComponent.js [hunk 2]
```diff
@@ -19,22 +76,55 @@ const ErrorConfig = {
...
+                {errorData?.mobile}
+              </a>
+            </span>
+          </p>
+        )}
         <button
           onClick={() => {
             props.goToHome();
           }}
+          style={styles.button}
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/core/src/components/ErrorComponent.js:127]: no need

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/core/src/components/ErrorComponent.js [hunk 3]
```diff
@@ -1,10 +1,67 @@
...
+    width: "153px",
+    borderRadius: "0px",
+    padding: "8px 24px",
+    color: "white",
+    cursor: "pointer",
+    backgroundColor: "#00796b",
+    border: "none",
+    fontSize: "16px",
+  },
+});
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/core/src/components/ErrorComponent.js:59]: no need

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/core/src/components/ErrorComponent.js [hunk 4]
```diff
@@ -19,22 +76,55 @@ const ErrorConfig = {
...
+                href={`tel:${errorData?.mobile}`}
+                style={{ ...styles.link, whiteSpace: "nowrap" }}
+                onMouseOver={(e) => (e.target.style.textDecoration = "underline")}
+                onMouseOut={(e) => (e.target.style.textDecoration = "none")}
+              >
+                {errorData?.mobile}
+              </a>
+            </span>
+          </p>
+        )}
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/core/src/components/ErrorComponent.js:122]: no need it works

**Pull request merged**